beamer/tests/agent/test_handlers.py

```diff
@@ -267,3 +278,62 @@ def test_handle_claim_made():
             claim_id=CLAIM_ID,
         )
     ]
+
+
+@patch("beamer.chain._withdraw")
+def test_maybe_claim_no_l1(patched_withdraw: MagicMock):
```

Comment:
I really prefer not to this. Is there not another way that you can check what you need, without resorting to patching private functions or making the function public?